install/jaxws/bin/ts.jte

```diff
@@ -79,12 +79,13 @@ endorsed.dirs.ri=${jaxws.home.ri}/modules/endorsed
 ###########################################################################
 # @jaxws.classes  should contain all Vendor specific jars/classes
 ###########################################################################
-jaxws.classes=${jaxws.lib}/webservices-api-osgi.jar${pathsep}${jaxws.lib}/jaxb-api-osgi.jar${pathsep}${jaxws.lib}/webservices-osgi.jar${pathsep}${jaxws.lib}/jaxb-osgi.jar${pathsep}${jaxws.lib}/gmbal.jar${pathsep}${jaxws.lib}/management-api.jar${pathsep}${jaxws.lib}/mimepull.jar${pathsep}${jaxws.lib}/ha-api.jar${pathsep}${jaxws.lib}/webservices-api.jar${pathsep}${jaxws.lib}/jakarta.xml.bind-api.jar${pathsep}${jaxws.lib}/jakarta.activation.jar
+jaxws.classes=${jaxws.lib}/../lib/resolver.jar${pathsep}${jaxws.lib}/jakarta.ejb-api.jar${pathsep}${jaxws.lib}/webservices-api-osgi.jar${pathsep}${jaxws.lib}/jaxb-api-osgi.jar${pathsep}${jaxws.lib}/webservices-osgi.jar${pathsep}${jaxws.lib}/jaxb-osgi.jar${pathsep}${jaxws.lib}/gmbal.jar${pathsep}${jaxws.lib}/management-api.jar${pathsep}${jaxws.lib}/mimepull.jar${pathsep}${jaxws.lib}/ha-api.jar${pathsep}${jaxws.lib}/webservices-api.jar${pathsep}${jaxws.lib}/jakarta.xml.bind-api.jar${pathsep}${jaxws.lib}/jakarta.activation.jar${pathsep}${jaxws.lib}/jakarta.servlet-api.jar${pathsep}${jaxws.lib}/FastInfoset.jar${pathsep}${jaxws.lib}/gmbal-api-only.jar${pathsep}${jaxws.lib}/ha-api.jar${pathsep}${jaxws.lib}/jakarta.annotation.jar${pathsep}${jaxws.lib}/jakarta.xml.soap-api.jar${pathsep}${endorsed.dirs}/jaxb-api.jar${pathsep}${jaxws.lib}/jaxb-impl.jar${pathsep}${jaxws.lib}/jaxb-xjc.jar${pathsep}${jaxws.lib}/jaxb-jxc.jar${pathsep}${jaxws.lib}/jaxb-core.jar${pathsep}${endorsed.dirs}/jaxws-api.jar${pathsep}${jaxws.lib}/jaxws-rt.jar${pathsep}${jaxws.lib}/jaxws-tools.jar${pathsep}${jaxws.lib}/jsr181-api.jar${pathsep}${jaxws.lib}/mail.jar${pathsep}${jaxws.lib}/management-api.jar${pathsep}${jaxws.lib}/mimepull.jar${pathsep}${jaxws.lib}/policy.jar${pathsep}${jaxws.lib}/resolver.jar${pathsep}${jaxws.lib}/saaj-impl.jar${pathsep}${jaxws.lib}/stax-ex.jar${pathsep}${jaxws.lib}/stax2-api.jar${pathsep}${jaxws.lib}/streambuffer.jar${pathsep}${jaxws.lib}/woodstox-core-asl.jar
```

Comment:
this is too much/contains duplicates. One should need only:
```
${jaxws.lib}/jakarta.xml.bind-api.jar${pathsep}${jaxws.lib}/jaxb-osgi.jar${pathsep}${jaxws.lib}/webservices-api-osgi.jar${pathsep}${jaxws.lib}/webservices-osgi.jar${pathsep}${jaxws.lib}/jakarta.activation.jar${pathsep}${jaxws.lib}/jakarta.annotation-api.jar${jaxws.lib}/jakarta.ejb-api.jar${pathsep}${pathsep}${jaxws.lib}/jakarta.mail.jar${pathsep}${jaxws.lib}/jakarta.servlet-api.jar${pathsep}${jaxws.lib}/mimepull.jar${pathsep}${jaxws.lib}/stax2-api.jar${pathsep}${jaxws.lib}/woodstox-core-asl.jar${pathsep}${jaxws.lib}/gmbal.jar${pathsep}${jaxws.lib}/management-api.jar${pathsep}${jaxws.lib}/ha-api.jar
```
(I hope I did not make a mistake :-) )


in general:
* `webservices-api-osgi.jar` is a subset of `webservices-api.jar` (one of these will go away at some point as the only major difference here should be inclusion/absence of OSGi headers) and already contains `jakarta.xml.soap-api.jar`, `jsr181-api.jar` (this name itself is incorrect), `jaxws-api.jar` (again, the name is wrong)
* `jakarta.xml.bind-api.jar` is the new name of `jaxb-api-osgi.jar` and `jaxb-api.jar`
* `webservices-osgi.jar` contains `jaxws-rt.jar`, `jaxws-tools.jar`, `policy.jar`, `saaj-impl.jar`, `streambuffer.jar`
* `gmbal-api-only.jar` is superseded in GF by `gmbal.jar`
* `jaxb-osgi.jar` contains `FastInfoset.jar`, `jaxb-impl.jar`, `jaxb-xjc.jar`, `jaxb-jxc.jar`, `jaxb-core.jar`, `stax-ex.jar`

As for the `resolver.jar` itself - it's part of Oracle/OpenJDK JDK 8; it should be required on JDK 8 implementations from other vendors (ie in the past, IBM JDK required this to be present) and should not be needed on JDK 9+

And few renames:
`jakarta.annotation.jar` -> `jakarta.annotation-api.jar`
`mail.jar`->`jakarta.mail.jar`

Reply:
btw: the comment above applies to GF (or metro as a whole). For standalone jax-ws impl, jars are different, see [the snapshot zip](https://ci.eclipse.org/metro/job/jax-ws-master-build/lastSuccessfulBuild/artifact/jaxws-ri/bundles/jaxws-ri/target/jaxws-ri.zip) for exact content